pyproject.toml

```diff
@@ -1,7 +1,36 @@
-[tool.isort]
-profile = "black"
-lines_after_imports = 2
-src_paths = ["kinto-remote-settings", "tests"]
+[tool.ruff]
+# Exclude a variety of commonly ignored directories.
+exclude = [
+    ".git",
+    "__pycache__",
+    ".venv",
+    "node_modules",
+]
+
+[tool.ruff.lint]
+preview = true
```

Comment:
Do we want this? Seems like it comes with less stability ([docs](https://docs.astral.sh/ruff/preview/))

Reply:
nope 👍